modular_skyrat/modules/modular_items/lewd_items/code/lewd_arousal/status_effects/climax.dm

```diff
@@ -8,15 +8,16 @@
 
 /datum/status_effect/climax/tick()
 	var/mob/living/carbon/human/affected_mob = owner
-	if(affected_mob.client?.prefs?.read_preference(/datum/preference/toggle/erp/sex_toy))
-		var/temp_arousal = -12
-		var/temp_pleasure = -12
-		var/temp_stamina = 15
+	if(!affected_mob.client?.prefs?.read_preference(/datum/preference/toggle/erp/sex_toy))
+		return
+	var/temp_arousal = -12
+	var/temp_pleasure = -12
+	var/temp_stamina = 15
```

Comment:
Those should be defines, rather than variables, as they don't actually get modified, they stay constant even at runtime.